lib/src/Pos/Util/Config.hs

```diff
@@ -38,7 +38,7 @@ embedYamlObject name marker parser = do
             Just dir -> return (dir </> name)
             Nothing  -> error $ toText $
                 "Could not find " ++ marker ++ " for path: " ++ srcFP
-    TH.qAddDependentFile path
+    -- TH.qAddDependentFile path
```

Comment:
Best to add comment:
> `qAddDependentFile` is disabled for _reason_. 
This is only used for `configuration.yaml` but still.

Reply:
Good that you are pointing this out. I had completely forgot about that.

Reply:
Alright, I've turned this into a patch, let's see if that's a stop-gap measure for now that works.